app/routes/users.js

```diff
@@ -83,6 +83,8 @@ export default Route.extend({
     //     });
     // }
       model(){
-        return this.store.findAll('user') ;
+        return this.store.findAll('user').then(results => results.filter((user)=>{
+          return user.get('isNew')==false
```

Comment:
Good. We can simplify this like, `user => !get(user, 'isNew')`
For single line functions, we don't need to explicitly return.